features/hsl.yml

```diff
@@ -0,0 +1,12 @@
+name: HSL
+description: The `hsl()` and `hsla()` CSS functions pick colors using hue, saturation and lightness channels.
+spec: https://drafts.csswg.org/css-color-4/#the-hsl-notation
+caniuse: css3-colors
+group: color-types
+status:
+  compute_from: css.types.color.hsl.alpha_parameter
```

Comment:
Some notes from the [initial PR](https://github.com/web-platform-dx/web-features/pull/1615)-

- @ddbeck suggested omitting this `compute_from`, as the alpha component was a key part of the story.  
- `mixed_type_parameters` was initially excluded to match Oklab, but the associated TODO was unrelated to `mixed_type_parameters`. I'm not sure why it was omitted there, and suggest including it here. 

However, that moves this to `baseline: false`, as Safari doesn't support mixed type parameters. So, I opted to set the `compute_from` to the `alpha_parameter` feature.

Reply:
OK that works. Except…

Comparing this with the linked caniuse entry (css3-colors), it appears that caniuse doesn't care about the alpha channel, so we make this feature appear rather newer than caniuse does (e.g., shipping in Edge 12 versus Edge 79). It's not a huge problem, but it is after our "don't worry about minor discrepancies before 2020" cutoff. I've filed https://github.com/mdn/browser-compat-data/issues/24191 on BCD to deal with the problem upstream.

In the mean time, I'd like to add something like this:

```suggestion
  compute_from: css.types.color.hsl.alpha_parameter
# TODO: For alignment with caniuse, add `hsla()` features when https://github.com/mdn/browser-compat-data/issues/24191 is resolved and update compute_from
```